package-lock.json

```diff
@@ -0,0 +1,10124 @@
+{
```

Comment:
We are using yarn, And its using yarn.lock. package-lock.json will be used if we will run commands using npm. So let's delete it